app/domain/errors.rb

```diff
@@ -354,10 +354,9 @@ module AuthnGcp
       )
 
       ResourceRestrictionNotFoundOrEmpty = ::Util::TrackableErrorClass.new(
-        msg:  "Resource restriction '{0-resource-restriction-name}' does not exists " \
-            "resource in JWT token. " \
-            "Verify that you configured the host with permitted restrictions. " \
-            "In case of Compute Engine token verify that you requested the token using 'format=full' " ,
+        msg:  "Resource restriction '{0-resource-restriction-name}' does not exist in the JWT. " \
```

Comment:
Did Shuli review? 
I would say not does not exists in JWT, but maybe `not found in any of the token claims`?